cmd/snap/cmd_known.go

```diff
@@ -31,11 +31,25 @@ import (
 	"github.com/jessevdk/go-flags"
 )
 
+type assertTypeName string
+
+func (n assertTypeName) Complete(match string) []flags.Completion {
+	names := asserts.TypeNames()
```

Comment:
you know what? this is just for completion. The only argument for calling `asserts.TypeNames()` is that it's faster (and that the import is already happening). And it's surprised both of you.

So I'm switching it to go through the client instead.